index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
If the content of your website is in German you should change the `lang` attribute to `de`. :) [See MDN for further information.](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)